packages/components/src/stepper/style.scss

```diff
@@ -24,11 +25,16 @@
 			margin: 0 4px 0 4px;
 		}
 
+		.woocommerce-spinner {
+			display: block;
+			margin-left: -$gap-smaller;
+			margin-right: $gap-smallest;
+		}
+
 		&.is-active,
 		&.is-complete {
-			.woocommerce-stepper__step-icon,
-			.components-spinner {
-				background: $muriel-hot-orange-700;
+			.woocommerce-stepper__step-icon {
+				background: $muriel-gray-900;
```

Comment:
Should we override this color in `profile-wizard/style.scss` so we can keep `$muriel-hot-orange-700` or should the Muriel component use this color by default?

Reply:
Just double checked the Components Figma, and it actually looks like this should be `$muriel-hot-blue-500` by default -- so I'll update that, and override with the gray in `profile-wizard/style.scss`.